MyBlog.API/Program.cs

```diff
@@ -1,8 +1,17 @@
+using Microsoft.EntityFrameworkCore;
+using MyBlog.Infrastructure;
+
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 
 builder.Services.AddControllers();
+
+var connection = builder.Configuration.GetConnectionString("DefaultConnection");
```

Comment:
if you can write by one line, write one line - you don`t need to split the string.